agent/pkg/utils/har.go

```diff
@@ -203,9 +202,11 @@ func NewResponse(response *api.GenericMessage) (harResponse *har.Response, err e
 
 	status := int(resDetails["status"].(float64))
 	if strings.HasPrefix(mimeType.(string), "application/grpc") {
-		status, err = strconv.Atoi(_status)
+		if _status != "" {
```

Comment:
why it is happen in GRPC and not before when we converting to int? (status := int(resDetails["status"].(float64)))

Reply:
@gadotroee the variable name is `_status` and not `status`. It's different, it comes from `:status` header.

Reply:
@gadotroee OK fixed the wrong variable name in `BuildHeaders` function.